### PR TITLE
fix bsc#1110300 by running disc proposal before bootloader

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
Port of commit
https://github.com/yast/skelcd-control-Kubic/commit/3a28ac79d59b50200755990812cad560513522cd

to CaaSP4

Signed-off-by: Jordi Massaguer Pla <jmassaguerpla@suse.de>

### CODE REVIEW

**Comment by teclator** [comment]
@jordimassaguerpla thnx for the fix, could you increase the version and add the changelog entry? thnx.

**Comment by jordimassaguerpla** [comment]
@teclator : done

File: package/skelcd-control-CAASP.changes [hunk 1]
```diff
@@ -1,3 +1,13 @@
+-------------------------------------------------------------------
+Thu Oct 11 08:35:10 UTC 2018 - jmassaguerpla@suse.com
+
+- fix bsc#1110300 by running disc proposal before bootloader
+    
+    Port of commit
+    https://github.com/yast/skelcd-control-Kubic/commit/3a28ac79d59b50200755990812cad560513522cd
+    
+    to CaaSP4
+
```
Comment (teclator) [inline package/skelcd-control-CAASP.changes:9]: just missed the version in the changelog

File: control/control.CAASP.xml [hunk 2]
```diff
@@ -385,11 +385,11 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
             <unique_id>inst_initial</unique_id>
             <enable_skip>no</enable_skip>
             <proposal_modules config:type="list">
-                <proposal_module>
```
Comment (teclator) [inline control/control.CAASP.xml]: Better now :+1:

**Review by teclator — APPROVED** [review]
LGTM, thnx @jordimassaguerpla !

**Comment by jordimassaguerpla** [comment]
I created also an IBS submit request for caasp4
https://build.suse.de/request/show/174412

**Pull request merged**